src/platform/forms/validations.js

```diff
@@ -348,6 +348,20 @@ function validateLength(
   };
 }
 
+function validateWhiteSpace(errors, input) {
+  if (typeof input !== 'undefined') {
+    if (
+      input.toString().startsWith(' ') ||
```

Comment:
This checks for space, but I was thinking a regex for all whitespace like [^\s]

Reply:
I figured there were cases where people have first names that include spaces. For instance: "Mary Jane".

Reply:
That's fine. The ^ is for starts with.

Reply:
\s is for any whitespace character